AssistantV1.playground/Pages/Introduction.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,85 @@
+//:# Watson Assistant V1
+
+import PlaygroundSupport
+
+//: This playground contains code examples for the Watson Assistant V1 service.
+
+// Enable support for asynchronous completion handlers
+PlaygroundPage.current.needsIndefiniteExecution = true
+
+//: To use the Watson Developer Cloud Swift SDK in your application, specify it in your `Cartfile`:
+//:```
+//:github "watson-developer-cloud/swift-sdk"
+//:```
+
+//:Then run the following command to build the dependencies and frameworks:
+//:```bash
+//:carthage update --platform iOS
+//:```
+
+import AssistantV1
+
+//:## Authentication
+
+let version = "2018-10-15"
+let apiKey = WatsonCredentials.AssistantV1APIKey
+
+let assistant = Assistant(version: version, apiKey: apiKey)
+
+//:## Service URL
+
+// Set the URL for the service endpoint if needed
+if let serviceURL = WatsonCredentials.AssistantV1URL {
+    assistant.serviceURL = serviceURL
+}
+
+//:## Error handling
+
+assistant.getWorkspace(workspaceID: "7777777-1111-dead-beef-ffffffffffff") {
+    response, error in
+
+    if let error = error {
+        switch error {
+        case let .http(statusCode, message, metadata):
+            switch statusCode {
+            case .some(404):
+                // Handle Not Found (404) exception
+                print("Not found")
+            case .some(413):
+                // Handle Request Too Large (413) exception
+                print("Payload too large")
+            default:
+                if let statusCode = statusCode {
+                    print("Error - code: \(statusCode), \(message ?? "")")
+                }
+            }
+        default:
+            print(error.localizedDescription)
+        }
+        return
+    }
+
+    guard let workspace = response?.result else {
+        print(error?.localizedDescription ?? "unknown error")
+        return
+    }
+
+    print(workspace)
+}
+
+//:### Pagination
```

Comment:
Instead of `Pagination`, how about we have 2 subsections called `Example 1` and `Example 2` for `getWorkspace()` and `listLogs()`, respectively?

Reply:
I used `Pagination` to match the heading in the API ref:

https://cloud.ibm.com/apidocs/assistant?code=java#pagination

Reply:
I noticed that we don't actually need to include this section. None of the languages have code examples for Pagination. I suppose it's fine to leave it in the playgrounds, but the `//:### Pagination` section header doesn't need to be parsed.

Reply:
Swift is showing leadership here!